pkg/admission/customresourcevalidation/authentication/validate_authentication_test.go

```diff
@@ -0,0 +1,189 @@
+package authentication
+
+import (
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func TestFailValidateAuthenticationSpec(t *testing.T) {
+	errorCases := map[string]struct {
```

Comment:
This is the "legacy" style of writing unit tests.  Look for the string `t.Run(tt.name, func(t *testing.T) {` to get an idea of what the expected generated test harness is supposed to look like (ex: `TestValidateGitHubIdentityProvider`).

The important part is that the harness is generated so that all tests look similar.

Reply:
I should have googled this more, I can fix this as a part of the other CR PR.